cloudfront/iiif.wellcomecollection.org/terraform/cf_behaviours.tf

```diff
@@ -196,35 +248,58 @@ locals {
     },
   ]
 
+  pdf_cover_behaviours = [
+    {
+      path_pattern     = "pdf-cover/*"
+      target_origin_id = "pdf_cover"
+      headers          = []
+      cookies          = null
```

Comment:
This can't be `null`.

Reply:
Made `"All"` - pdf_cover page will never care about cookies hence the null

Reply:
I think you could make this "none".